r/.Rprofile

```diff
@@ -1 +1,3 @@
 source("renv/activate.R")
+for (f in list.files('R', '.R$', full.names = TRUE)) source(f)
+require(dplyr)
```

Comment:
Why put this in .Rprofile?

Reply:
To make it faster to setup the working environment. Just open the project and have everything up in memory and all the worker methods accesible

Reply:
The typical way to do this would be to `@export` any function that you want available and then install the pipeline as a package (`ctrl+shift+B`) then just `library(pipeline)`. We can do this too if you're finding it works well